source/newcommitter.md

```diff
@@ -295,11 +295,11 @@ This is the followup email after the new committer has accepted the invitation
 
     Just as before you became a committer, participation in any ASF community
     requires adherence to the ASF Code of Conduct:
-      https://www.apache.org/foundation/policies/conduct.html
+      https://www.apache.org/foundation/policies/conduct.html.
```

Comment:
The trailing . might make it hard to simply click on the link unless there is some metadata around it.